src/middlewared/middlewared/plugins/interface/configure.py

```diff
@@ -210,7 +222,7 @@ def configure(self, data, aliases, wait_dhcp=False, options=None):
             self.logger.debug('Starting dhclient for {}'.format(name))
             self.middleware.call_sync('interface.dhclient_start', data['int_interface'], wait_dhcp)
 
-        if osc.IS_FREEBSD:
+        if not linux:
```

Comment:
Keeping osc.IS_FREEBSD makes it easy for us to spot places to remove code blocks later on if needed.

Just personal opinion, no need to change this time/PR

Reply:
I agree, i'll change back since there is no reason for me to make it inconsistent.

Reply:
Changed it back to be consistent with how we do it everywhere else.